### DISCUSSION TITLE
[RFC FS-1080 Discussion] - Float32 without dot

### DISCUSSION THREAD

**Discussion by gdziadkiewicz** [fsharp/fslang-design#414] [Language and core library RFC discussions]

**Comment by abelbraaksma** [comment]
Linking the RFC here for ease of reference: https://github.com/fsharp/fslang-design/blob/master/FSharp-5.0/FS-1080-float32-without-dot.md

**Comment by abelbraaksma** [comment]
Suggested update on RFC: https://github.com/fsharp/fslang-design/pull/416
